### PR TITLE
Updated copy to be more legible and clear.

### PR DESCRIPTION
**Pull request by jzhang729**
---
Please check the following conditions for your PR.

- [  ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ✔️  ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by jzhang729** [comment]
Can I get some help with fixing these errors generated from paconn validate?

`➜  TractionGuest git:(dev) paconn validate --api-def ./apiDefinition.swagger.json
{
  "error": {
    "code": "SwaggerCertificationFailedWithErrors",
    "message": "Swagger certification failed with errors. Please validate again after fixing following errors: Error : info : The 'contact' property is required.\r\nError : The 'x-ms-connector-metadata' property is required."
  }
}
400 Client Error: Bad Request for url: https://api.powerapps.com/providers/Microsoft.PowerApps/objectIds/291d6a5d-45c6-4552-884f-085fa03758b0/validateApiSwagger?api-version=2016-11-01&enableConnectorCertificationRules=true`

**Comment by dougelliott** [comment]
@jzhang729 I've responded to your questions about the validation errors via email, just a heads up :)

**Comment by jzhang729** [comment]
> @jzhang729 I've responded to your questions about the validation errors via email, just a heads up :)

Thanks @dougelliott. Fixed the validation errors now.

**Comment by jzhang729** [comment]
@dougelliott I guess I should wait until this PR is merged and squashed down into a single commit before I submit this into ISV studio?

Screen Shot 2021-02-09 at 12 02 45 PM

File: certified-connectors/TractionGuest/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/TractionGuest/apiDefinition.swagger.json:682]: @jzhang729 Can you please let us know what documentation you are referring to add metadata?
The `publisher` and `stackOwner` properties should be added to `apiProperties.json` file. Please remove it from Swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by dougelliott** [comment]
@jzhang729 Correct :) Once pending items are taken care of within the PR, we'll handle squashing and merging for you. And once that's done, you can move forward with submitting the update within ISV Studio.

On that note, Srikanth has left a comment for your review above. Feel free to let us know if there are any questions on that.

File: certified-connectors/TractionGuest/apiDefinition.swagger.json
Comment (jzhang729) [reply certified-connectors/TractionGuest/apiDefinition.swagger.json]: Hi @sriyen-msft . I think I was looking at other connectors like Envoy and Azure AD as examples.

Based on your feedback, I just pushed a new commit with those changes.

**Comment by jzhang729** [comment]
@sriyen-msft Any ETA on when we can get this reviewed? (cc @dougelliott)

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**
